src/node/splitid_endpoints.h

```diff
@@ -0,0 +1,643 @@
+        const auto in = params.get<SignRPC::In>();
+        auto cfg = consensus->get_latest_configuration_unsafe();
+        std::vector<ccf::NodeId> nids;
+        for (auto& [nid, _] : cfg)
+        {
+          nids.push_back(nid);
+        }
+
+        auto ctx = get_splitid_ctx(context);
+        SignRPC::Out r;
+
+        if (!args.tx.ro(ctx->current_identity)->get(0).has_value())
+        {
+          throw std::runtime_error("no split identity available (yet)");
+        }
+
+        auto rss = args.tx.rw(ctx->resharing_sessions);
+        auto sss = args.tx.rw(ctx->sampling_sessions);
+        if (rss->size() != 0 || sss->size() != 0)
+        {
+          return make_error(
+            HTTP_STATUS_SERVICE_UNAVAILABLE,
+            ccf::errors::InternalError,
+            "A sampling or resharing session is in progress");
+        }
+
+        SigningSession<ccf::NodeId> ns(
+          nids, in.message, in.defensive, in.app_id);
+        auto vals = args.tx.template rw<ccf::Values>(ccf::Tables::VALUES);
+        auto id = ccf::get_next_id(vals, ccf::NEXT_SIGNING_SESSION_ID);
+        auto signing_session = args.tx.rw(ctx->signing_sessions);
+        signing_session->put(id, ns);
+        return make_success(SignRPC::Out({id}));
+      }
+      catch (std::exception& ex)
+      {
+        return make_error(
+          HTTP_STATUS_BAD_REQUEST, ccf::errors::InternalError, ex.what());
+      }
+    };
+
+    endpoints
+      ->make_endpoint(
+        "splitid/sign", HTTP_POST, json_adapter(splitid_sign), no_auth_required)
+      .set_forwarding_required(endpoints::ForwardingRequired::Always)
+      .install();
+
+    auto splitid_get_sig = [&consensus, &context](
+                             auto& args, const nlohmann::json& params) {
+      try
+      {
+        const auto in = params.get<GetSigRPC::In>();
+        auto cfg = consensus->get_latest_configuration_unsafe();
+        std::vector<ccf::NodeId> nids;
+        for (auto& [nid, _] : cfg)
+        {
+          nids.push_back(nid);
+        }
+
+        auto tx = args.tx.rw(get_splitid_ctx(context)->signing_sessions);
+        auto s = tx->get(in.session_id);
+        if (!s.has_value())
+        {
+          throw std::logic_error("No such session.");
+        }
+        auto& session = s.value();
+        if (session.signature.empty())
+        {
+          return make_error(
+            HTTP_STATUS_NOT_FOUND,
+            ccf::errors::ResourceNotFound,
+            fmt::format(
+              "Signature for session #{} not available (yet).", in.session_id));
+        }
+        else
+        {
+          tx->remove(in.session_id);
+          return make_success(GetSigRPC::Out({session.signature}));
+        }
+      }
+      catch (std::exception& ex)
+      {
+        return make_error(
+          HTTP_STATUS_BAD_REQUEST, ccf::errors::InternalError, ex.what());
+      }
+    };
+
+    endpoints
+      ->make_endpoint(
+        "splitid/get-signature",
+        HTTP_POST,
```

Comment:
```suggestion
        "splitid/signature",
        HTTP_GET,
```

Should look at the other endpoints HTTP methods as well. I think that (mostly) the URLs are nouns? In which case we should use `GET`/`POST` as appropriate.